### PR TITLE
environments: Regenerate manifest

### PR DESCRIPTION
**Pull request by awgreene**
Adds OLM Metrics.

### CODE REVIEW

**Comment by awgreene** [comment]
Unlike the example PR, changes to the `jsonnetfile.lock.json` were not made after running `./build.sh`.
The changes introduced to the `observatorium-template.yaml ` file were manually added.

File: jsonnetfile.lock.json [hunk 1]
```diff
@@ -1,125 +1,114 @@
...
+        {
+            "name": "telemeter",
+            "source": {
+                "git": {
+                    "remote": "https://github.com/openshift/telemeter",
+                    "subdir": "jsonnet/telemeter"
+                }
+            },
+            "version": "8dae8893776e237688d7ff39821265a4f467f213"
+        },
```
Comment (LiliC) [inline jsonnetfile.lock.json]: You just need to commit this change only, and then run the `./build.sh`.

Comment (awgreene) [reply jsonnetfile.lock.json]: @LiliC I updated the `version`, but there was no value to replace the existing `sum` with.

Comment (awgreene) [reply jsonnetfile.lock.json]: After attempting to merge the change I got this error:
> jb: error: failed to install packages: checksum mismatch for telemeter. Expected ks15FK8j+RCu6QpmgI6Ofiy/RZ8eLQVss4VNo/IWQts= but got tLjALazakKQ6Z0VtG74rX/TLJo4Jh7lvN6fkXHWJxNc=

I'll update the sum to match it.

Comment (LiliC) [reply jsonnetfile.lock.json]: What do you mean merge the change?

Comment (awgreene) [reply jsonnetfile.lock.json]: Sorry - I updated the PR to include `tLjALazakKQ6Z0VtG74rX/TLJo4Jh7lvN6fkXHWJxNc=` as the sum.

**Comment by LiliC** [comment]
cc @squat do you mind having a look at this, thanks :)

**Comment by squat** [comment]
Ah wait, actually this PR is an exact duplicate of https://github.com/observatorium/configuration/pull/133. We can close this as it’s already done :)

**Pull request closed without merging**